src/sidebar.js

```diff
@@ -200,9 +197,9 @@ class Sidebar extends React.Component {
   }
 
   render() {
-    let sidebarStyle = styles.sidebar,
+    let sidebarStyle = Object.assign({}, styles.sidebar, { zIndex: this.props.zIndex + 1 }),
```

Comment:
I was under the impression that the babel polyfill is not included by default. You have to explicitly require it.

I would probably do a better job if I would write it today but when I started ES6 was all still pretty new to me. You're welcome to help improve it :+1: